src/imgui_ex/vcImGuiSimpleWidgets.cpp

```diff
@@ -367,3 +367,33 @@ void vcIGSW_Markdown(vcState *pProgramState, const char *pMarkdownText)
 
   ImGui::Markdown(pMarkdownText, udStrlen(pMarkdownText), config);
 }
+
+bool vcIGSW_DegreesScalar2D(const char *pLabel, udDouble2 *pData)
+{
+  udDouble2 tmp = *pData;
+  tmp.x = UD_RAD2DEG(tmp.x);
+  tmp.y = UD_RAD2DEG(tmp.y);
```

Comment:
I could be wrong, but I think you can collapse this down to `udDouble2 tmp = UD_RAD2DEG(*pData);`? Similarly, the inverse in the if statement below.